function-images/tests/producer-consumer/client/client.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"google.golang.org/grpc"
+
+	pb "github.com/MBaczun/producer-consumer/prodcon"
+)
+
+func main() {
+	strings := flag.Int("s", 1, "Number of strings to send")
+	address := flag.String("addr", "localhost", "Server IP address")
+	clientPort := flag.Int("pc", 3031, "Client Port")
```

Comment:
keep in mind for future that knative demands a specific port to send requests to